Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -1691,20 +1691,20 @@ script:
       secret: false
     - default: false
       description: List of IP addresses.
-      isArray: false
+      isArray: true
       name: ip_list
       required: false
       secret: false
     - default: false
       description: Name of the endpoint group.
-      isArray: false
+      isArray: true
```

Comment:
The name of the group, operating system etc are also lists?
Also, why did we change this?

Reply:
Yes, it should be a list https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-api/cortex-xdr-apis/response-actions/scan-endpoints.html
We also did it in `get_endpoints_command` function